orte/mca/ess/base/ess_base_std_orted.c

```diff
@@ -111,6 +112,10 @@ int orte_ess_base_orted_setup(char **hosts)
     char *param;
     hwloc_obj_t obj;
     unsigned i, j;
+    orte_topology_t *t;
+    opal_list_t transports;
```

Comment:
Other than generating an unused variable warning, these two lines won't hurt but are not really part of this change. You might want to remove them.